packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx

```diff
@@ -194,6 +196,10 @@ export class CalendarMonth extends BaseComponent<ICalendarMonthProps, {}> {
     }
   }
 
+  private _onSelectMonthKeyDown = (index: number): (ev: React.KeyboardEvent<HTMLElement>) => void => {
+    return (ev: React.KeyboardEvent<HTMLElement>) => this._onKeyDown(() => this._onSelectMonth(index), ev);
```

Comment:
Do you want both `ENTER` and `SPACE` because otherwise `onClick` will work for both mouse and `SPACE` since its a `<button/>`. I ask because I recently fixed a semi-related bug where buttons were firing their handlers twice -- https://github.com/OfficeDev/office-ui-fabric-react/pull/4662.

If we continue with this approach, I do think we can reduce this function signature by at least eliminating one closure, but I need to try myself with code.

Reply:
For the SPACE issue, good call, that was an unintentional bug. Should be fixed now.